entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/UserEntityProvider.java

```diff
@@ -241,7 +241,9 @@ public void updateEntity(EntityReference ref, Object entity, Map<String, Object>
             edit.setEmail(u.getEmail());
             edit.setFirstName(u.getFirstName());
             edit.setLastName(u.getLastName());
-            edit.setPassword(u.getPassword());
+            if (u.getPassword() != null) {
```

Comment:
Also, createEntity has a similar line, does that need an update?
`edit.setPassword(user.getPassword());`